pkgs/development/libraries/libcamera/default.nix

```diff
@@ -44,13 +54,19 @@ stdenv.mkDerivation {
 
     # cam integration
     libevent
+    libdrm
+
+    # hotplugging
+    systemd
 
     # lttng tracing
     lttng-ust
 
     gtest
   ];
 
+  FONTCONFIG_FILE = fontsConf;
```

Comment:
Probably should go next to the other env vars. And a comment why it is needed would be useful.

Reply:
I'm not even sure if this is the proper way to get rid of the fontconfig errors, it's just what I found was being done in other packages. It only gets rid of the "no config file" errors anyway, there are still like a thousand lines of "no writable cache".

Reply:
Yeah, this is the proper “fix”. I am mostly interested what needs fonts in libcamera. It is typically needed by GUI tests or sometimes docs generation.

Reply:
Probably docs yes, unless gstreamer requires it.

Reply:
What commands precede the warning?

Reply:
Seems indeed to be to the documentation. Build log says
```
[256/378] Generating doxygen with a custom commandap_ipa_proxyxyrialization_test.cpp.oK[Kerializer_test.cpp.o.o
/build/libcamera-1db1e31/src/libcamera/base/file.cpp:49: warning: no matching class member found for
  libcamera::File::MapFlag::NoOption
<a few more similar warnings but this looks like it's interspersed here from another command running in parallel>
sh: latex: not found
error: Problems running latex. Check your installation or look for typos in _formulas.tex and check _formulas.log!
sh: dvips: not found
error: Problems running dvips. Check your installation!
Fontconfig error: No writable cache directories
Fontconfig error: No writable cache directories
Fontconfig error: No writable cache directories
...
```

The documentation is built with sphinx, which does mention LaTeX as one backend (for pdf rendering) right on its landing page. There is no option for the documentation format and the `meson.build` for the documentation specifically requests html output so my best guess is that this is sphinx probing the environment to determine which backends are actually usable.